### PR TITLE
Añadidos mis proyectos

### PR DESCRIPTION
**Pull request by pepdbm7**
calculator y pasapalabra

### CODE REVIEW

File: staff/pep-delbano/pre-course/calculator/stylescalculadora.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.page {
+  width: 700px;
+}
+
+header {
+  width: 70%;
+  margin: auto; /*para centrar dcha izda*/
+  margin-top: 30px;
```
Comment (LluisBagur) [inline staff/pep-delbano/pre-course/calculator/stylescalculadora.css]: - Another way to write it:
margin: 30px 0 80px 0;

with this we can modify everything in one line.

File: staff/pep-delbano/pre-course/calculator/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</head>
+
+<body>
+  <div class="page">
+
+    <header>
+      <h1>Calculator</h1>
+    </header>
+
+    <form class="calculadora"name="calculadora">
```
Comment (scrafy) [inline staff/pep-delbano/pre-course/calculator/index.html]: Its better to use a div as label, The form is used to send data to backend.

File: staff/pep-delbano/pre-course/calculator/READMI.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## CALCULATOR
```
Comment (JudithGo) [inline staff/pep-delbano/pre-course/calculator/READMI.md]: The title of the file should be README.md

File: staff/pep-delbano/pre-course/pasapalabra/READMI.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+## PASAPALABRA
```
Comment (JudithGo) [inline staff/pep-delbano/pre-course/pasapalabra/READMI.md]: Todos los archivos README están mal escritos.

File: staff/pep-delbano/pre-course/calculator/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+## CALCULATOR
+
+------------
+
+This project is a calculator machine made with just **_HTML5_, _CSS3_ y _JS_**. 
+
+![alt=3logos](./html-css-js.jpeg)
```
Comment (gbaste) [inline staff/pep-delbano/pre-course/calculator/README.md]: Hi Pep, bad link image.

File: staff/pep-delbano/pre-course/calculator/index.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+<body>
+  <div class="page">
+
+    <header>
+      <h1>Calculator</h1>
+    </header>
+
+    <label class="calculadora"name="calculadora">
+
+    <input class="resultado" type="text" name="resultado">
```
Comment (gbaste) [inline staff/pep-delbano/pre-course/calculator/index.html:21]: you should not mix languages, preferable English

File: staff/pep-delbano/pre-course/pasapalabra/mainpasapalabra.js [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+      {num: 26, letter: "z", answer: "zen", status: 0, question: ("CON LA Z. Escuela de budismo que busca la experiencia de la sabiduría más allá del discurso racional") },
+  ];
+
+
+var pos = 0;  //recuento de pendientes por responder!
+var count = 0;  //recuento de contestadas!
+var letra;
+var respuesta;
+var totalTiempo = 120;
+var t;
```
Comment (gbaste) [inline staff/pep-delbano/pre-course/pasapalabra/mainpasapalabra.js:37]: Your variables should be more semantic. var t? var res?

File: staff/pep-delbano/cv/cv.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (gbaste) [inline staff/pep-delbano/cv/cv.html]: Define a title for your HTML document
https://www.w3schools.com/tags/tag_title.asp

**Pull request merged**